src/PlantsViewer.php

```diff
@@ -6,18 +6,6 @@
 
 class PlantViewHelper
 {
-    public static function displaySinglePlant (Plant $plant): string
-    {
-        $output = "<div class='individual_plants'>";
-        $output .= "<h1>$plant->name</h1>";
-        $output .= "<p>$plant->family</p>";
-        $output .= "<img src='$plant->image' />";
-        $output .= "<p class='description'>$plant->description</p>";
-        $output .= "</div>";
-
-        return $output;
-    }
-
     public static function displayAllPlants (array $plants): string
```

Comment:
You should consider what would happen if there were no plants in the collection, if the $plants array is empty you should return some kind of basic message. That would also mean your PlantsViewerTest would need a failure test